tests/utils.js

```diff
@@ -745,6 +745,12 @@ module.exports = {
     });
   },
 
+  deleteDocNative: async id => {
+    const doc = await module.exports.getDocNative(id);
+    doc._deleted = true;
+    return module.exports.saveDocNative(doc);
```

Comment:
Very optional es6 "alternative":
```suggestion
    return module.exports.saveDocNative({...doc, _deleted: true});
```

...one thing I like better about this (and what caused me to look at the lines in detail in the first place, is that is generates a new object rather than mutating the original object. Not critical in this case, but also a decent practice to get into. `</ immutable-salespitch>`